studygroups/models.py

```diff
@@ -75,25 +78,29 @@ class StudyGroup(models.Model):
     location = models.ForeignKey('studygroups.Location')
     location_details = models.CharField(max_length=128)
     facilitator = models.ForeignKey(User)
-    start_date = models.DateTimeField() # Both day and time could be captured by start_date - eg: start Wednesday 1 July at 19:00 implies regular meetings on Wednesdays at 19:00
+    start_date = models.DateTimeField() # start_date also implies regular meeting day & time. Ex. Wednesday 1 July at 19:00 implies regular meetings on Wednesday at 19:00
     end_date = models.DateTimeField()
-    day = models.CharField(max_length=128, choices=zip(calendar.day_name, calendar.day_name)) #TODO remove this field
-    time = models.TimeField() #TODO remove this field
     duration = models.IntegerField() # meeting duration in minutes
     timezone = models.CharField(max_length=128)
-    max_size = models.IntegerField()
+    max_size = models.IntegerField() #TODO remove this field
     description = models.TextField()
     created_at = models.DateTimeField(auto_now_add=True)
 
+    def day(self):
```

Comment:
True, I think it used to be used for calculating dates also, but probably not anymore